examples/benchmarks/lstm_tagger.py

```diff
@@ -16,10 +16,18 @@
 import tqdm
 import numpy.random
 from timeit import default_timer as timer
-from thinc.api import Model, Config, registry, chain, list2padded, with_array
+from thinc.api import (
+    Model,
+    Config,
+    registry,
+    chain,
+    list2padded,
+    with_array,
+    with_padded,
+)
```

Comment:
When this happens due to `black` formatting, we'd typically spread the `import` statements over 2 lines "manually"